Lombiq.BaseTheme/Views/GoogleTag.cshtml

```diff
@@ -1,29 +1,34 @@
-@model Lombiq.BaseTheme.ViewModels.GoogleTagViewModel
+@model dynamic
 
 @using Microsoft.AspNetCore.Http.Features;
 @using Microsoft.Extensions.Hosting;
-@using OrchardCore.Environment.Shell
+@using Lombiq.BaseTheme.ViewModels
 
 @inject IHostEnvironment HostEnvironment
 
 @{
+    var viewModel = Model as GoogleTagViewModel ?? new GoogleTagViewModel 
+    {
+        GoogleTagPropertyId = Model.GoogleTagPropertyId,
+        CookieDomain = Model.CookieDomain,
+    };
```

Comment:
This and the rest of the code block except for `trackingConsentFeature` should be below in the `if` instead.